lrpar/doc/src/quickstart.md

```diff
@@ -0,0 +1,94 @@
+# Quickstart Guide
+
+This is a short guide to get you started. For a more detailed explanation about
+what's going on, please refer to the [tutorial](tutorial/index.html).
+
+We begin by creating a `Cargo.toml` file and load the necessary dependencies:
+
+```toml
+[package]
+name = "project"
+version = "0.0.1"
+authors = ["<authors>"]
+
+[[bin]]
+doc = false
+name = "project"
+
+[build-dependencies]
+lrlex = { path="<path to lrlex>" }
```

Comment:
When we make a release (which hopefully won't be long!), we have to remember to change this to `lrlex = "0.1"` and so on.